flask_app.py

```diff
@@ -261,24 +271,25 @@ def check_identity_or_auth(request):
         resp = make_response(render_template('auth.html', redditauthlink=reddit_link, discordauthlink=discord_link))
         return (resp, None)
 
-    headers = {"Authorization": "bearer " + access_token, 'User-agent': 'CFB Risk Orders'}
-    response = requests.get(REDDIT_ACCOUNT_URI, headers=headers)
-    if response.status_code == 401:
+
+    if auth_site == REDDIT_SITE:
+        headers = {"Authorization": "bearer " + access_token, 'User-agent': 'CFB Risk Orders'}
+        response = requests.get(REDDIT_ACCOUNT_URI, headers=headers)
+    elif auth_site == DISCORD_SITE:
         headers = {"Authorization": "Bearer " + access_token, 'User-agent': 'CFB Risk Orders'}
         response = requests.get(DISCORD_ACCOUNT_URI, headers=headers)
-        if response.status_code == 401:
-            log.error(f"{access_token},401 Error from CFBR API")
-            reddit_link = make_reddit_authorization_url()
-            discord_link = make_discord_authorization_url()
-            resp = make_response(render_template('auth.html', redditauthlink=reddit_link, discordauthlink=discord_link))
-            return (resp, None)
+      
```

Comment:
It's good to have an `else:` clause to fall back on and give us a good error in case there's some case where `auth-site` is not set or not one of the supported values, otherwise you're going to get a NameError below that `response` is not defined.